src/widget/wsearchrelatedtracksmenu.cpp

```diff
@@ -1,10 +1,18 @@
 #include "widget/wsearchrelatedtracksmenu.h"
 
+#include <QScreen>
+
 #include "track/track.h"
+#include "util/math.h"
 #include "util/qt.h"
+#include "util/widgethelper.h"
 
 namespace {
 
+constexpr double kMaxMenuToAvailableScreenWidthRatio = 0.2; // 20%
```

Comment:
I presume this is an arbitrary cutoff that you think looks good? If so, please note in the comment that this is arbitrary.

Reply:
Done.

I also removed the other arbitrary lower bound that is not strictly needed and only complicates the code. Less code is better code.